<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="1" failed="1" ignored="0" total="3" passed="1">
  <reporter-output>
  </reporter-output>
  <suite name="N11 Test Suite" duration-ms="40354" started-at="2017-10-12T23:15:20Z" finished-at="2017-10-12T23:16:00Z">
    <groups>
    </groups>
    <test name="LoginTest" duration-ms="40354" started-at="2017-10-12T23:15:20Z" finished-at="2017-10-12T23:16:00Z">
      <class name="tests.LoginTests">
        <test-method status="PASS" signature="setup()[pri:0, instance:tests.LoginTests@6c25e6c4]" name="setup" is-config="true" duration-ms="10844" started-at="2017-10-13T02:15:20Z" finished-at="2017-10-13T02:15:31Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method status="PASS" signature="invalidLoginTest_InvalidUserNameInvalidPassword()[pri:0, instance:tests.LoginTests@6c25e6c4]" name="invalidLoginTest_InvalidUserNameInvalidPassword" duration-ms="14503" started-at="2017-10-13T02:15:31Z" description="Invalid Login Scenario with wrong username and password." finished-at="2017-10-13T02:15:45Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_InvalidUserNameInvalidPassword -->
        <test-method status="SKIP" signature="invalidLoginTest_EmptyUserEmptyPassword()[pri:1, instance:tests.LoginTests@6c25e6c4]" name="invalidLoginTest_EmptyUserEmptyPassword" duration-ms="4839" started-at="2017-10-13T02:15:46Z" description="Invalid Login Scenario with empty username and password." finished-at="2017-10-13T02:15:50Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at pages.LoginPage.verifyLoginPassword(LoginPage.java:40)
	at tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_EmptyUserEmptyPassword -->
        <test-method status="FAIL" signature="invalidLoginTest_EmptyUserEmptyPassword()[pri:1, instance:tests.LoginTests@6c25e6c4]" name="invalidLoginTest_EmptyUserEmptyPassword" duration-ms="6641" started-at="2017-10-13T02:15:51Z" description="Invalid Login Scenario with empty username and password." finished-at="2017-10-13T02:15:58Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [WRONG MESSAGE FOR FAILURE!] but found [Bu alanın doldurulması zorunludur.]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertEqualsImpl(Assert.java:134)
	at org.testng.Assert.assertEquals(Assert.java:115)
	at org.testng.Assert.assertEquals(Assert.java:189)
	at org.testng.Assert.assertEquals(Assert.java:199)
	at pages.LoginPage.verifyLoginPassword(LoginPage.java:40)
	at tests.LoginTests.invalidLoginTest_EmptyUserEmptyPassword(LoginTests.java:57)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.retryFailed(Invoker.java:1014)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1215)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:283)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:120)
	at org.apache.maven.surefire.booter.ForkedBooter.invokeProviderInSameClassLoader(ForkedBooter.java:386)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:323)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:143)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalidLoginTest_EmptyUserEmptyPassword -->
        <test-method status="PASS" signature="teardown()[pri:0, instance:tests.LoginTests@6c25e6c4]" name="teardown" is-config="true" duration-ms="2272" started-at="2017-10-13T02:15:58Z" finished-at="2017-10-13T02:16:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- tests.LoginTests -->
    </test> <!-- LoginTest -->
  </suite> <!-- N11 Test Suite -->
</testng-results>
